bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -28,7 +28,7 @@ <h3 class="c-item-title">{{ title }}</h3>
           {% if src_still %}
           <source srcset="{{ src_still }}" media="(prefers-reduced-motion: reduce)" loading="lazy" width="280" height="280"></source>
           {% endif %}
-          <img src="{{ src }}" loading="lazy" width="280" height="280">
+          <img src="{{ src }}" loading="lazy" width="280" height="280" alt="{{ alt }}">
```

Comment:
I'm not sure on the right answer here, but is `alt` text on the `img` fallback enough to satisfy screen readers that encounter the `<picture>` element?

Reply:
My research and testing suggest this is the best way. Chrome didn't idenify it as an image but it did read the alt text. It ended up sounding like a subheading because the heading is first in the source order.